modules/ossm-troubleshooting-injection.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-injection_{context}"]
+= Troubleshooting sidecar injection
+
+{ProductName} relies on a proxy sidecar within the application’s pod to provide Service Mesh capabilities to the application.
+
+Unlike how the upstream Istio community configures proxy sidecars using namespace labels, {ProductName} does not automatically inject proxy sidecars to pods.  You must opt in to sidecar injection.
+
+== Troubleshooting Istio sidecar injection
+
+Check to see if automatic injection is enabled in the Deployment file for your application.  If automatic injection for the Envoy proxy is enabled, there should be a `sidecar.istio.io/inject:"true"` annotation in the Deployment file.
```

Comment:
The annotation actually needs to be in `spec.template.metadata.annotations` for `Deployment` resources, as the istio mutating webhook gets applied to pods, not deployments.